src/modules/Dashboard/components/ActivityAssignDrawer/ActivityAssignDrawer.tsx

```diff
@@ -72,6 +72,9 @@ export const ActivityAssignDrawer = ({
   ...rest
 }: ActivityAssignDrawerProps) => {
   const { t } = useTranslation('app', { keyPrefix: 'activityAssign' });
+  const { featureFlags } = useFeatureFlags();
+  const enableActivityAssignFlag = featureFlags.enableActivityAssign;
```

Comment:
Same thing here. It's safe to assume `ActivityAssignDrawer` isn't accessible if `enableActivityAssign` is not enabled.